enterprise_gateway/itests/test_python_kernel.py

```diff
@@ -84,9 +84,9 @@ def test_get_deploy_mode(self):
         result = self.kernel.execute("sc.getConf().get('spark.submit.deployMode')")
         self.assertRegexpMatches(result, '(cluster|client)')
 
-    def test_get_host_address(self):
-        result = self.kernel.execute("print(sc.getConf().get('spark.driver.host'))")
-        self.assertRegexpMatches(result, 'itest')
+    def test_get_hostname(self):
+        result = self.kernel.execute("import subprocess; subprocess.check_output(['hostname'])")
+        self.assertRegexpMatches(result, os.environ['ITEST_HOSTNAME_PREFIX'] + "*")
 
```

Comment:
Thanks for updating the test method.  Since this test is now nolonger dependent on Spark, we should probably move these into the xxxKernelBaseTestCase classes.